src/libraries/Microsoft.Extensions.DependencyInjection/tests/Microsoft.Extensions.DependencyInjection.Tests.csproj

```diff
@@ -2,18 +2,20 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
-    <RootNamespace>Microsoft.Extensions.DependencyInjection</RootNamespace>
+    <DefineConstants>$(DefineConstants);SKIP_FAILING</DefineConstants>
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
-    <Reference Include="Microsoft.Extensions.DependencyInjection.Specification.Tests" />
-    <Reference Include="Microsoft.Extensions.DependencyInjection" />
-    <Reference Include="Microsoft.Extensions.Logging.Abstractions" />
-    <Reference Include="Newtonsoft.Json" />
+    <Compile Include="*.cs" />
+    <Compile Include="DI.Specification.Tests/Fakes/**/*.cs" />
+    <Compile Include="DI.Specification.Tests/*.cs" />
+    <Compile Include="../../Common/tests/Extensions/TestingUtils/Microsoft.AspNetCore.Testing/**/*.cs" />
```

Comment:
Let's not include common source by wild-card.  Please select each file and set the link metadata.